package.json

```diff
@@ -7,20 +7,22 @@
     "build:analyze": "ANALYZE_BUNDLE=true yarn build:single",
     "build:demo": "lerna run build:demo --stream",
     "build:single": "utils/scripts/scoped-npm-command.js --script build",
+    "build:storybook": "build-storybook -s ./utils/styleguide/public -o ./demo/storybook",
     "format": "prettier-package-json --write && yarn format:package_json --write && yarn format:js --write && yarn format:md --write",
     "format:all": "prettier-package-json --list-different && yarn format:package_json --list-different && yarn format:js --check && yarn format:md --check",
     "format:js": "prettier --loglevel warn '{packages,utils}/**/*.{js,ts,tsx}' '!packages/**/dist/**'",
     "format:md": "prettier --loglevel warn 'packages/**/*.md'",
     "format:package_json": "lerna exec -- prettier-package-json",
     "postinstall": "lerna bootstrap",
     "lint": "yarn lint:css && yarn lint:js && yarn lint:md",
-    "lint:css": "stylelint '{packages,utils}/**/*.{js,ts,tsx}' '!packages/**/*.spec.{js,ts,tsx}' '!packages/**/dist/**'",
-    "lint:js": "eslint packages/*/src/ utils/ --ext js,ts,tsx --max-warnings 0",
-    "lint:md": "markdownlint README.md docs/*.md packages/*/examples/*.md packages/*/src/**/*.md packages/*/README.md",
+    "lint:css": "stylelint '{packages,utils,stories}/**/*.{js,ts,tsx}' '!packages/**/*.spec.{js,ts,tsx}' '!packages/**/dist/**'",
+    "lint:js": "eslint packages/*/src/ utils/ .storybook/ --ext js,ts,tsx --max-warnings 0",
+    "lint:md": "markdownlint README.md docs/*.md stories/**/*.md packages/*/examples/*.md packages/*/src/**/*.md packages/*/README.md",
     "new": "utils/scripts/new.js",
     "prepare": "yarn build",
     "start": "utils/scripts/scoped-npm-command.js --script start",
     "start:demo": "live-server demo",
+    "start:storybook": "start-storybook -s ./utils/styleguide/public -p 6006",
```

Comment:
Dev ergonomics aren't ideal, but good enough for now. Let's keep an eye our for when storybook is `build:demo` and `start:demo` and we move styleguidist over to temporary `build:styleguidist` and `start:styleguidist` until retirement.

Reply:
For consistency I was planning on replacing:
- `yarn start` (becomes `start-storybook` command)
- `yarn build:demo` (becomes `build-storybook` command)

and removing these storybook-specific tasks once the final package is moved over. The nice thing is we should see the build-times gradually reduce as each package is moved over.